app/apps/client/gameCenter/tabBarButton.js

```diff
@@ -0,0 +1,17 @@
+import { TabBar } from '../../../ui-utils/client';
+import { APIClient } from '../../../utils';
+
+(async function() {
+	const { games } = await APIClient.get('apps/games');
+	// TODO Need to filter the games enabled by the admin only
```

Comment:
we need to perform the intersection between enabled apps and installed components here so that users can only see the enabled games in the game center.